src/main/java/hudson/plugins/parameterizedtrigger/DefaultParameterValuesActionsTransform.java

```diff
@@ -1,11 +1,6 @@
 package hudson.plugins.parameterizedtrigger;
 
-import hudson.model.AbstractProject;
-import hudson.model.Action;
-import hudson.model.ParametersAction;
-import hudson.model.ParameterDefinition;
-import hudson.model.ParameterValue;
-import hudson.model.ParametersDefinitionProperty;
+import hudson.model.*;
```

Comment:
if I was a maintainer of this I would reject such a change.
You may get lucky...